From a0ccb4970ef73af2fb5f8b5366d1b4738c99ab57 Mon Sep 17 00:00:00 2001
From: Enrico Scholz <enrico.scholz@informatik.tu-chemnitz.de>
Date: Sat, 21 Nov 2009 14:12:41 +0100
Subject: [PATCH 5/6] fixed unitialized variable

Reported by Nikola Pajkovsky <npajkovs AT redhat.com>:

  Problem shows up only when you compiled xmlrpc with nss and try to
  connect to server with wrong certificate.
---
 lib/curl_transport/curltransaction.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/lib/curl_transport/curltransaction.c b/lib/curl_transport/curltransaction.c
index 630f318..efcbcb0 100644
--- a/lib/curl_transport/curltransaction.c
+++ b/lib/curl_transport/curltransaction.c
@@ -509,6 +509,7 @@ curlTransaction_create(xmlrpc_env *               const envP,
         curlTransactionP->curlSessionP = curlSessionP;
         curlTransactionP->userContextP = userContextP;
         curlTransactionP->progress     = progress;
+	curlTransactionP->curlError[0] = '\0';
 
         curlTransactionP->serverUrl = strdup(serverP->serverUrl);
         if (curlTransactionP->serverUrl == NULL)
-- 
1.6.5.2

